src/pages/profile/index.js

```diff
@@ -279,21 +276,19 @@ const ViewProfile = () => {
                   placeholder="Tell us about yourself, your professional and educational highlights to date..."
                   spellCheck="false"
                   value={userProfile.bio}
-
                 ></textarea>
                 <small>0/300</small>
               </div>
               <div class="button-container">
-                { allowedToEdit ? (
+                {allowedToEdit ? (
                   <button
                     type="button"
                     class="button offwhite"
                     onClick={handleClick}
                   >
                     Edit
                   </button>
-                  ) : null
-                }
+                ) : null}
```

Comment:
If you want to show something or not, use an && statement.

If you want to show something or something else, use an if statement